php72u.spec

```diff
@@ -1792,6 +1792,9 @@ exit 0
 
 
 %changelog
+* Mon Oct 24 2019 Seth Ryder <seth@sethryder.com> - 7.2.24-1
+- Re-add pid file used by logrotate to fix logrotate issue
```

Comment:
Can you adjust this changelog entry to say it's updating to the latest upstream version?

Reply:
sorry about that. just pushed the updated changelog entry.

Reply:
No worries, that's what reviews are for.  :grin: